content/styles/02-molecules/hero.css

```diff
@@ -0,0 +1,13 @@
+.hero h2, 
+.hero h3 {
+  color: var(--white);
+}
+
+.hero h2 {
+  font-size: 8rem;
+  margin-bottom: 0;
+}
+
+.hero .button {
```

Comment:
Hum?

Reply:
eyes of tiger!!! (but already fix when you was writtin this ^^)